cache/private/private_topic.go

```diff
@@ -67,9 +67,10 @@ func UpdateCensusTopic(ctx context.Context, serviceAuthToken string, topicClient
 func UpdateDataTopics(ctx context.Context, serviceAuthToken string, topicClient topicCli.Clienter) func() []*cache.Topic {
 	return func() []*cache.Topic {
 		var topics []*cache.Topic
+		processedTopics := make(map[string]bool)
 
-		// get the root topics subtopics (e.g. economy) from dp-topic-api
-		rootTopics, err := topicClient.GetSubtopicsPrivate(ctx, topicCli.Headers{ServiceAuthToken: serviceAuthToken}, cache.RootTopicID)
+		// get root topics from dp-topic-api (e.g. economy) from dp-topic-api
+		rootTopics, err := topicClient.GetTopicPrivate(ctx, topicCli.Headers{}, cache.RootTopicID)
```

Comment:
You still need to pass the ServiceAuthToken here.

Reply:
Recommend testing this running in publishing mode, testing against the publishing mount.

Reply:
I'm not sure this comment is true either anymore. Or that you retrieve rootTopics - you're retrieving the rootTopic no?